src/lib/ThumbnailsSidebar.js

```diff
@@ -192,6 +253,40 @@ class ThumbnailsSidebar {
         pageNumberEl.textContent = `${pageNumber}`;
         return pageNumberEl;
     }
+
+    /**
+     * Sets the currently selected page
+     *
+     * @param {number} pageNumber - The page number to set to selected
+     * @return {void}
+     */
+    setCurrentPage(pageNumber) {
+        const parsedPageNumber = parseInt(pageNumber, 10);
+        if (!parsedPageNumber || parsedPageNumber < 1 || parsedPageNumber > this.pdfViewer.pagesCount) {
```

Comment:
How generic to we want to make this? This wouldn't work for multi page images.

Reply:
yeah, I was debating this as well, but I figured that would be a refactor for later, unless you feel this would be better served to be done upfront